src/components/popover-surface/popover-surface.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../style/functions';
+
+/**
+ * @prop --popover-surface-width: Width of the popover surface. Defaults to `auto`
+ * @prop --popover-header-background-color: Background color of popover header, deafutls to `--contrast-300`.
+ * @prop --popover-body-background-color: Background color of popover body, deafutls to `--contrast-100`.
+ * @prop --popover-footer-background-color: Background color of popover footer, deafutls to `--contrast-500`.
```

Comment:
These should be moved to `popover.scss` otherwise they will not appear in the documentation